docs/enterprise/prometheus.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Prometheus
+sidebarDepth: 1
+description: Use Prometheus as a metrics data store.
+---
+
+# Prometheus Metrics
+
+The Pomerium Enterprise Console uses Prometheus as a metrics collection back-end. You can configure Pomerium and the Console to talk to an existing Prometheus server, or configure the embedded Prometheus backend.
+
+## External Prometheus
+
+1. In the Pomerium `config.yaml` define the `metrics_address` key to a network interface and port. For example:
+
+   ```yaml
+   metrics_address: localhost:9999
```

Comment:
We have to be careful with this option.  The address needs to be remotely resolvable or an IP address if using embedded prometheus:

```yaml
metrics_address: myhost.my.subdomain.com:9999
```

If using external prometheus, the hostname/IP is not needed.  We default to all interfaces if there is only a port specified: `:9999`.

Reply:
This section is specifically for external. I'm looking to you for insight on how to enable embedded in the empty section below.

Reply:
> If using external prometheus, the hostname/IP is not needed. We default to all interfaces if there is only a port specified: :9999.

This is only true if the "external" Prometheus is running on the same host, right?

Reply:
https://github.com/pomerium/pomerium/pull/2390/commits/08ef65e72a7c92ad406833fa5c69cce456afe3e8

Reply:
it's true regardless of where external prometheus is running